coverage/coverage_gen.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+lines=$(ps -ef | grep /usr/local/bin/coverage | wc -l)
+echo $lines
+
+
```

Comment:
Please add a comment for below code logic For example:  send a SIGINT signal to all coverage process.

Reply:
ok